### PR TITLE
Patch for letter links

### PR DESCRIPTION
**Pull request by baileythegreen**
## Author: 

- Bailey Harrington

@fmichonneau After your commit to and merge of #207, the site isn't rendering correctly. I have found a missing close quote in the CSS link you altered in `_layouts/page.html` and `_layouts/glossary-rtl.html`, which seems to be the only cause. I have checked this in my the version rendered from my fork, and once I fixed it, it looks good.

## The Carpentries version of the site with CSS not working
The Carpentries version of the site

## My version of the site with CSS working
https://baileythegreen.github.io/glosario
My version of the site

### CODE REVIEW

**Review by fmichonneau — APPROVED** [review]

**Comment by fmichonneau** [comment]
I was waiting for the site to build and render after I merged your PR but my son woke up before I could see the result. Thanks for catching and fixing this!

**Comment by baileythegreen** [comment]
@fmichonneau 🤩 Not going to lie, I caught it because I wanted to see my work displayed on someone else's site, and then panicked when it wasn't working!

**Pull request merged**
